nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -39,6 +42,36 @@ namespace nav2_rviz_plugins
 {
 using nav2_util::geometry_utils::orientationAroundZAxis;
 
+// Define the row numbers for navigation_data_table_
+enum class NavigationDataTableRows {
+  // Waypoint number
+  kWaypoint,
```

Comment:
why do you have "k" at the beginning, could you maybe remove them and probably rename them?

Reply:
Because ROS tier one packages use `ament_lint_common` which uses google style guide. Google style guide says enums follow that format. 

Reference: https://google.github.io/styleguide/cppguide.html#Enumerator_Names